deployment/dynatrace-azure-logs.sh

```diff
@@ -268,7 +268,6 @@ fi
 check_arg --deployment-name "$DEPLOYMENT_NAME" "$DEPLOYMENT_NAME_REGEX"
```

Comment:
Same question as in AWS, wouldn't line 170 fail the deployment without the param?